src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -196,26 +196,39 @@ $sidenav-icon-size: 19px;
   &-settings {
     @include font-styles(nav-toggles);
 
-    .nav-menu-setting:first-child:not(:only-child) {
-      margin-right: $nav-space-between-elements;
-
-      @include nav-in-breakpoint() {
-        margin-right: 0;
-      }
-    }
-
     @include breakpoint-only-largenav() {
       margin-left: $nav-space-between-elements;
     }
 
-    @include nav-in-breakpoint {
-      padding-top: 0;
+    .nav-menu-setting {
+      display: flex;
+      align-items: center;
+      color: var(--color-nav-current-link);
+      margin-left: 0;
+
+      &:first-child:not(:only-child) {
+        margin-right: $nav-space-between-elements;
+
+        @include nav-in-breakpoint() {
+          margin-right: 0;
+        }
+      }
+
+      @include nav-dark() {
+        color: var(--color-nav-dark-current-link);
+      }
+
+      @include nav-in-breakpoint() {
+        &:not([data-previous-menu-children-count="0"]) {
```

Comment:
I think this line is for the `.nav-menu-settings`, as `data-previous-menu-children-count` is applied on it.

This is basically to not apply a border-top, in case the nav is in a state, where it has no items above the setting links.